relayer/src/main.rs

```diff
@@ -139,24 +144,72 @@ impl Synchronizer {
     fn get_bitcoin_block_hash_by_height(
         &self,
         height: u64,
-    ) -> Result<String, bitcoincore_rpc::Error> {
+    ) -> Result<String, Box<dyn std::error::Error>> {
         let block_from_bitcoin_node = self.bitcoin_client.get_block_header_by_height(height)?;
 
         Ok(block_from_bitcoin_node.block_hash().to_string())
     }
 }
 
+// TODO: cleanup this code
+async fn init_contract(bitcoin_client: &BitcoinClient, near_client: &NearClient, init_height: u64) {
```

Comment:
Cleanup this function

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/97/commits/3a66bbe0ffdde88ed30c0a033e58d691075459d2